src/App.js

```diff
@@ -4,7 +4,7 @@ import { Switch, Route } from 'react-router-dom';
 
 // Custom Imports
 import { Home, List, AddItem, Footer } from './components/component.index';
-
+import TestComponent from './Testing/test.component';
```

Comment:
Hi Team. Not sure, but thought I would ask if including this component on a different branch be more appropriate?